### PR TITLE
[14.0][ADD] purchase_work_acceptance_tier_validation

### PR DESCRIPTION
**Pull request by kittiu**
cc @newtratip @ps-tubtim

### CODE REVIEW

File: purchase_work_acceptance_tier_validation/tests/test_purchast_work_acceptance_tier_validation.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestHrExpenseTierValidation(common.TransactionCase):
```
Comment (ps-tubtim) [inline purchase_work_acceptance_tier_validation/tests/test_purchast_work_acceptance_tier_validation.py]: Please edit the class name.

Comment (kittiu) [reply purchase_work_acceptance_tier_validation/tests/test_purchast_work_acceptance_tier_validation.py]: Fixed.

**Review by ps-tubtim — APPROVED** [review]

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Pull request merged**